yarn.lock

```diff
@@ -2672,7 +2672,7 @@
     "@types/minimatch" "*"
     "@types/node" "*"
 
-"@types/google-protobuf@^3.10.0", "@types/google-protobuf@^3.7.2":
+"@types/google-protobuf@^3.10.0":
```

Comment:
this file needs to be reverted, you probably need to init your `e` submodule, i think our `webapps.e` uses the `3.7.2` version